packages/perseus/src/widgets/interactive-graphs/mafs-graph.test.tsx

```diff
@@ -21,17 +25,41 @@ function getBaseMafsGraphProps(): MafsGraphProps {
         box: [400, 400],
         step: [1, 1],
         gridStep: [1, 1],
-        snapStep: [1, 1],
         markings: "graph",
+        containerSizeClass: "small",
+        showTooltips: false,
+        labels: ["x", "y"],
+        dispatch: () => {},
+        state: {
```

Comment:
Question: Was state added here so that the custom state objects in the tests can overwrite this information?